js/index.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+	//waits for document to fully load to start function
+	$('form').submit(addCity);
+
+	function addCity (event) {
+		event.preventDefault();
+
+		var varcity = $('#city-type').val();
+
```

Comment:
This should be `var city`, not `var varcity` - I think perhaps you just misread the rubric requirement here. We want the name of the variable to just be `city`. It has no effect on your code though, since you consistently name the variable `varcity` everywhere else.